### PR TITLE
Fix Qt error about socket, and D-Bus accesses

### PR DESCRIPTION
**Pull request by hfiguiere**
I have these fixes in my PR for beta as well.

### CODE REVIEW

**Comment by hfiguiere** [comment]
The errors it removes are:

```
(mscore:2): dbind-WARNING **: 15:29:58.944: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
QtSingleCoreApplication: listen on local socket failed, QLocalServer::listen: Name error
[2:22:0118/153016.335386:ERROR:object_proxy.cc(616)] Failed to call method: org.gnome.SessionManager.Inhibit: object_path= /org/gnome/SessionManager: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
[2:22:0118/153016.335434:ERROR:power_save_blocker_x11.cc(335)] No response to Inhibit() request!
[2:22:0118/153016.393081:ERROR:object_proxy.cc(616)] Failed to call method: org.gnome.SessionManager.Uninhibit: object_path= /org/gnome/SessionManager: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
[2:22:0118/153016.393100:ERROR:power_save_blocker_x11.cc(408)] No response to Uninhibit() request!
```

File: org.musescore.MuseScore.yaml [hunk 1]
```diff
@@ -28,8 +28,9 @@ finish-args:
   # Allow loading/saving files from anywhere
   # (portals don’t work yet)
   - --filesystem=host
-  # Allow other instances to see lockfiles
-  - --env=TMPDIR=/var/tmp
+  # MuseScore want access to the session manager
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.a11y.Bus
```
Comment (TingPing) [inline org.musescore.MuseScore.yaml]: Its fine to have a workaround here but this sounds like a Qt bug. The portal provides access to a11y itself, raw access should never be needed, so something weird is going on.

File: org.musescore.MuseScore.yaml [hunk 2]
```diff
@@ -28,8 +28,9 @@ finish-args:
   # Allow loading/saving files from anywhere
   # (portals don’t work yet)
   - --filesystem=host
-  # Allow other instances to see lockfiles
-  - --env=TMPDIR=/var/tmp
+  # MuseScore want access to the session manager
+  - --talk-name=org.gnome.SessionManager
```
Comment (TingPing) [inline org.musescore.MuseScore.yaml]: There is also a portal for this, so this should be fixed upstream.

File: org.musescore.MuseScore.yaml [hunk 3]
```diff
@@ -28,8 +28,9 @@ finish-args:
   # Allow loading/saving files from anywhere
   # (portals don’t work yet)
   - --filesystem=host
-  # Allow other instances to see lockfiles
-  - --env=TMPDIR=/var/tmp
```
Comment (TingPing) [inline org.musescore.MuseScore.yaml]: Why was this removed?

Comment (hfiguiere) [reply org.musescore.MuseScore.yaml]: For
```
QtSingleCoreApplication: listen on local socket failed, QLocalServer::listen: Name error
```

This actually delayed the startup.

Comment (TingPing) [reply org.musescore.MuseScore.yaml]: Better question: Does single instance work after this removal?

Comment (hfiguiere) [reply org.musescore.MuseScore.yaml]: I have no idea how to test this.

**Comment by hfiguiere** [comment]
Rebased on the 3.4 branch (to be merged too)

File: org.musescore.MuseScore.yaml (hunk 3 above)
Comment (TingPing) [reply org.musescore.MuseScore.yaml]: launch the application twice..?

Comment (hfiguiere) [reply org.musescore.MuseScore.yaml]: ah right. and it fails. I'll figure something out. (#50 doesn't include this)

File: org.musescore.MuseScore.yaml (hunk 1 above)
Comment (hfiguiere) [reply org.musescore.MuseScore.yaml]: I have removed the a11y bus, as it was a user error here.

File: org.musescore.MuseScore.yaml (hunk 3 above)
Comment (hfiguiere) [reply org.musescore.MuseScore.yaml]: Removed this....

**Comment by hfiguiere** [comment]
Only the SessionManager is left.
